### PR TITLE
[plg_user_joomla]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_user_joomla
### Testing Instructions

code review

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 054056e902e454f738abc0b842f7eac192c801ba<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10510">issues.joomla.org/joomla-cms/10510</a>.</sub>

File: plugins/user/joomla/joomla.xml [hunk 1]
```diff
@@ -19,35 +19,38 @@
...
 				</field>
 
-				<field name="mail_to_user"
+				<field
+					name="mail_to_user"
 					type="radio"
-					class="btn-group btn-group-yesno"
-					default="1"
 					label="PLG_USER_JOOMLA_FIELD_MAILTOUSER_LABEL"
+					class="btn-group btn-group-yesno"
```
Comment (andrepereiradasilva) [inline plugins/user/joomla/joomla.xml]: the order should be name, type, label, description

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on e82da4416f4a09b478fbd54c7e81f3f74ac05917<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10510">issues.joomla.org/joomla-cms/10510</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e82da4416f4a09b478fbd54c7e81f3f74ac05917<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10510">issues.joomla.org/joomla-cms/10510</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10510">issues.joomla.org/joomla-cms/10510</a>.</sub>

**Pull request merged**